src/components/toggle/toggle.scss

```diff
@@ -0,0 +1,86 @@
+@import '../../styles/variables';
+
+.kui-theme--light {
+  --color-toggle-on: #{$color-primary-2};
+  --color-toggle-on-bar: #{rgba($color-primary-2, 0.3)};
+  --color-toggle-off: #{$color-bg-dark-9};
+  --color-toggle-off-bar: #{rgba($color-bg-dark-9, 0.3)};
+}
+
+.kui-theme--dark {
+  --color-toggle-on: #{$color-primary};
+  --color-toggle-on-bar: #{rgba($color-primary, 0.3)};
+  --color-toggle-off: #{$color-light};
+  --color-toggle-off-bar: #{rgba($color-light, 0.3)};
+}
+
+.pipeline-toggle {
```

Comment:
I see that this naming was adopted from the classes from `metadata-code` ( which I agree also shouldn't be named as `pipeline-(something)` - as per my other comment, given that this is a general react component, please rename this to a classname that is scoped within the toggle button. Naming this `pipeline-` will make future debugging harder and more confusing.